source/administration/object-management/object-versioning.rst

```diff
@@ -266,6 +266,105 @@ enable versioning on only a prefix or subset of objects in a bucket.
 Objects created prior to enabling versioning have a 
 ``null`` :ref:`version ID <minio-bucket-versioning-id>`.
 
+
+Exclude a Prefix From Versioning
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can exclude certain prefixes from versioning using the MinIO :mc:`mc` CLI.
+This is useful for Spark/Hadoop workloads or others that initially create objects
+with temporary prefixes.
+
+- Use the :mc-cmd:`mc version enable` command with the ``--excluded-prefixes``
+  option:
+
+  .. code-block:: shell
+     :class: copyable
+
+     mc version enable --excluded-prefixes ALIAS/BUCKET "prefix1, prefix2"
+
+  - Replace ``ALIAS`` with the :mc:`alias <mc alias>` of a configured 
+    MinIO deployment.
+
+  - Replace ``BUCKET`` with the 
+    :mc-cmd:`target bucket <mc version ALIAS>` for which the prefixes should
+    be excluded from versioning.
+
+Something about wildcards and globbing
+    
+You can exclude up to 10 prefixes for each bucket. To add or remove excluded prefixes,
+repeat your ``mc version enable`` command with an updated list. The new list of
+prefixes replaces the previous one.
+
+To view the currently excluded prefixes, use the :mc-cmd:`mc version enable`
+command with the ``--json`` option. The ``ExcludedPrefixes`` property contains a list
```

Comment:
You can simplify this, I think to
> To view the excluded prefixes, use :mc-cmd:`mc version enable` with the  :mc-cmd:`~mc  version enable --json` option.

When we use `~` at the start of the :mc-cmd:, it only displays the last element of the command, so we can link directly to the flag in the docs.

Reply:
Except this doesn't work, because --json is a global flag. So ignore this.